apps/Gruntfile.js

```diff
@@ -318,6 +318,8 @@ module.exports = function (grunt) {
         {pattern: 'lib/**/*', watched: false, included: false, nocache: true},
         {pattern: 'build/**/*', watched: false, included: false, nocache: true},
         {pattern: 'static/**/*', watched: false, included: false, nocache: true},
+        {pattern: 'lib/p5play/p5.js'},
+        {pattern: 'lib/p5play/p5.play.js'},
```

Comment:
You could import them where you need them using the webpack script-loader. See https://github.com/webpack/script-loader.